common-framework-utils/src/main/java/org/wso2/carbon/integration/common/utils/mgt/ServerConfigurationManager.java

```diff
@@ -521,6 +521,21 @@ public void copyToComponentDropins(File jar) throws IOException, URISyntaxExcept
         FileManager.copyJarFile(jar, lib);
     }
 
+    /**
+     * /**
+     * Copy Jar file to server component/dropins when carbonHome provided
```

Comment:
Comments need to start with Capital letter and end with '.'

Fix in other places as well